test/e2e/data/infrastructure-metal3/cluster-template-centos/kustomization.yaml

```
@@ -0,0 +1,7 @@
+bases:
+- ../bases/cluster-with-kcp.yaml
+- ../bases/ippool.yaml
+- ../bases/md.yaml
```

If you put a `kustomize.yaml` file in `bases` with this in it:
```yaml
resources:
- cluster-with-kcp.yaml
- ippool.yaml
- md.yaml
```
we could change this and also get rid of the load restrictor when building.

```suggestion
resources:
- ../base
```

(I changed `bases -> resources` since `bases` is deprecated in kustomize. They basically work the same now.)

---

At some point I think we might not need ippool.yaml that was the case in the upgrade (test workload template)

---

In that case we should have a separate base for the IP pool. Something like this:
```
├── bases
│   ├── cluster
│   │   ├── cluster-with-kcp.yaml
│   │   ├── kustomization.yaml
│   │   └── md.yaml
│   └── ippool
│       ├── ippool.yaml
│       └── kustomization.yaml
└── overlays
    ├── centos
    │   └── kustomization.yaml
    └── ubuntu
        └── kustomization.yaml
```